src/universal/modules/teamDashboard/containers/ArchiveTeamContainer/ArchiveTeamContainer.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react';
+import {withRouter} from 'react-router';
+import ArchiveTeam from 'universal/modules/teamDashboard/components/ArchiveTeam/ArchiveTeam';
+
+const ArchiveTeamContainer = (props) => {
+  const {
+    teamId,
+    teamName,
+    router
+  } = props;
+
+  return (
```

Comment:
this is where `showConfirmationField` should live, that way we separate the state from the view layer.